tests/queries/0_stateless/01866_split_by_regexp.reference

```diff
@@ -0,0 +1,6 @@
+['a','ba','ba','b']
```

Comment:
```suggestion
['a','ba','ba','b']
['a','ba','ba','b']
```

Reply:
Is this applied? I saw errors in last check.